js/form-validation-check.js

```diff
@@ -34,12 +36,32 @@ pristine.addValidator(formForValidation.querySelector('.text__description'), val
 pristine.addValidator(textHashtags, testInfo, 'Поле не может быть пустое.');
 
 function testInfo (value) {
-  return  value !== '';
+  return  value === '';
 }
+function submitFormAndClosingEditor (closeEdit){
+  formForValidation.addEventListener('submit',(evt)=>{
+    evt.preventDefault();
+    const isValid = pristine.validate();
+    if (!isValid){
+      const formData = new FormData(evt.target);
+      fetch('https://25.javascript.pages/got',
```

Comment:
И отправить не понятно куда, вот правильный адрес:
```suggestion
      fetch('https://25.javascript.pages.academy/kekstagram',
```